web/themes/custom/pineapple/sass/base/typography/_headings.scss

```diff
@@ -28,7 +19,7 @@ h2 {
   font-family: $header-font-family;
 }
 
-h3 {
+.u-text-h3 {
```

Comment:
just h3 is fine, remove the 'u-text-' on this and the classes below

Reply:
done